### DISCUSSION TITLE
TuringCourses/major_mandatory/cognitive_neuroscience/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#197] [General]
# TuringCourses/major_mandatory/cognitive_neuroscience/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_mandatory/cognitive_neuroscience/

**Comment by 626lt** [comment]
从给分来看，唐老师的给分似乎存在大小年，22级的给分普遍不是很好，也许因为复习课透了很多的题，最后考试的判卷比较严格，平时分似乎也没有给的很慷慨，~~也是贡献了笔者的新史低绩点~~。今年的评分是40%（论文展示）+ 60%期末考试。论文展示一般的同学也许只有85分左右，以后应该也会延续这样的考核方式，应当重视这一点。

**Comment by ken0yuan** [comment]
认可大小年，此外，请高度重视历年卷，唐老师每年的卷子有可能改都不改。今年是作业和考试的评价方法
